elements/pfe-primary-detail/demo/index.html

```diff
@@ -69,15 +69,80 @@
       .ansible-styles .pfe-primary-detail__toggle--active {
         font-weight: bold;
       }
+
+      pfe-primary-detail .focus-styles:focus,
```

Comment:
Why the light DOM styles here?  Are you testing that the styles from the component override these?  Since they match, it will be hard to see.  If these are for testing, can you change them to different values to validate the override is working?

Reply:
We removed this